vimeo-networking/src/main/java/com/vimeo/networking/model/Connection.java

```diff
@@ -57,6 +57,12 @@
     @GsonAdapterKey("viewable_total")
     public int viewableTotal;
 
+    @GsonAdapterKey("new_total")
+    public int newTotal;
+
+    @GsonAdapterKey("unread_total")
+    public int unreadTotal;
```

Comment:
since we have getters below for these fields, can we make them package local or protected?